### ISSUE TITLE
In structure.py, when vpt_type is shallow, is there something wrong with the code? 

### ISSUE THREAD

**Issue by tb2-sy** [sagizty/VPT#3]
num_tokens = x.shape[1] should be placed after x = torch.cat((x, Prompt_Tokens), dim=1)

**Comment by sagizty** [comment]
Totally right ! It's a debugging mistake, and I will correct it right now.

**Comment by sagizty** [comment]
Problem solved, please check the latest version.

**Comment by tb2-sy** [comment]
> Problem solved, please check the latest version.

Thank you so much for sharing!
